### PR TITLE
NCDA - color indicator for MUAC

### PR DESCRIPTION
**Pull request by anvmn**
#908

### CODE REVIEW

**Comment by anvmn** [comment]
Peek 2023-10-15 15-06

**Comment by anvmn** [comment]
@mariano-dagostino please review

File: client/src/elm/Measurement/View.elm [hunk 1]
```diff
@@ -3099,6 +3099,46 @@ ncdaFormInputsAndTasks language currentDate zscores personId person config form
...
+                inputsAndTasks =
+                    List.map inputsAndTasksForSign signs
+            in
+            ( List.concatMap Tuple.first inputsAndTasks
+            , List.concatMap Tuple.second inputsAndTasks
+            )
+
+        NCDAStepNutritionBehavior ->
+            let
+                breasdtfeedingSign =
```
Comment (mariano-dagostino) [inline client/src/elm/Measurement/View.elm]:
```suggestion
                breastfeedingSign =
```

**Review by mariano-dagostino — APPROVED** [review]
@anvmn minor typo.

**Pull request merged**